### PR TITLE
Add simple UI for visualizing Result data.

### PR DESCRIPTION
**Pull request by wlynch**
This folder contains a test HTTP server for rendering Results in a very basic
UI.

This is **not** intended to be a replacement UI for
[dashboard](https://github.com/tektoncd/dashboard) - this is only here for
local testing to help visualize result data.

This is only intended for testing, so testing rigor and polish are not
strictly enforced.

### CODE REVIEW

File: tools/simpleui/main.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	}
+
+	r := mux.NewRouter()
+	r.HandleFunc("/", u.home)
+	r.HandleFunc("/{namespace}", u.results)
+	r.HandleFunc("/{namespace}/results/{name}", u.records)
+	http.Handle("/", r)
+
+	hostname, _ := os.Hostname()
+	log.Printf("Running on http://%s:8080\n", hostname)
```
Comment (XinruZhang) [inline tools/simpleui/main.go:78]: Tiny error here: the hostname doesn't work for me: 

> 2021/02/01 16:12:11 Running on http://xinruzhang-macbookpro.roam.corp.google.com:8080

Using `localhost` instead solved the problem. Is there any other function can fit this requirement?

**Review by XinruZhang — COMMENTED** [review]
Looks great!

File: tools/simpleui/main.go (hunk 1 above)
Comment (wlynch) [reply tools/simpleui/main.go:78]: Done.

**Pull request merged**